Trappist/src/Promact.Trappist.DomainModel/Models/Test/Test.cs

```diff
@@ -5,10 +5,9 @@ namespace Promact.Trappist.DomainModel.Models.Test
 {
     public class Test : BaseModel
     {
-        [Required] 
-        [MaxLength(150, ErrorMessage = "Test Name length Should be less than 150")] 
-        //Test Name Should be alpha-numeric and should accept some special characters as defined
-        [RegularExpression("^[a - zA - Z0 - 9_@ $#%&*^{}+;:<>()-]*$",ErrorMessage ="Enter A Valid Test Name")]
+        [Required]
+        [MaxLength(150, ErrorMessage = "Test Name length Should be less than 150")]
```

Comment:
ErrorMessage is written in wrong english. In any statement only 1st character is capital. except for few cases.